src/System.Security.Cryptography.Algorithms/src/Resources/Strings.resx

```diff
@@ -79,6 +79,9 @@
   <data name="Arg_CryptographyException" xml:space="preserve">
     <value>Error occurred during a cryptographic operation.</value>
   </data>
+  <data name="Cryptography_BadData" xml:space="preserve">
+    <value>Bad Data.</value>
```

Comment:
What should the developer do who reads this message? Can we give them a hint at what they should change, or check on?